packages/firebase_ml/android/src/main/java/io/flutter/plugins/firebaseml/FirebaseMLPlugin.java

```diff
@@ -40,21 +50,86 @@ public void onAttachedToEngine(@NonNull FlutterPluginBinding flutterPluginBindin
     channel.setMethodCallHandler(this);
   }
 
+  @RequiresApi(api = Build.VERSION_CODES.N)
   @Override
-  public void onMethodCall(@NonNull MethodCall call, @NonNull Result result) {
+  public void onMethodCall(@NonNull MethodCall call, @NonNull final Result result) {
```

Comment:
For a large method like this with isolated pieces of logic it is preferred to break it up via private helper methods.

https://screenshot.googleplex.com/JSV0MchbpZc
go/java-practices/methods#length

Reply:
Done. In fact, new class ModelManager is cleated (similar to Detector in ML Vision) to handle these method calls.